charts/k8ssandra/values.yaml

```diff
@@ -588,6 +588,11 @@ monitoring:
 cleaner:
   image: k8ssandra/k8ssandra-cleaner:e6c3702701ca
 
+# k8ssandra-client provides CLI utilities, but also certain functions such as 
+# upgradecrds that allow modifying the running instances
+client:
+  image: burmanm/k8ssandra-client:8772293f
```

Comment:
I am not sure how I feel about pulling in k8ssandra-client in this PR. I have only looked at it briefly. It looks like a lot of really good stuff. With that said, I have a couple concerns. First, it is scope creep. Secondly and more importantly, I think there ought to be some discussion, design docs, etc. on what we expect for a CLI client before we go ahead and introduce one into the project. Clearly you are in the best position to drive that effort 😄